src/components/CaptureDetailTooltip.js

```diff
@@ -21,9 +24,20 @@ const CaptureDetailTooltip = ({ capture, showCaptureClick }) => {
       marginLeft: '12px',
     },
   }));
+  const [speciesName, setSpeciesName] = React.useState();
+  const { speciesList } = useContext(SpeciesContext);
 
-  const CaptureDetailTooltipStyles = CaptureDetailTooltipUseStyles();
+  React.useEffect(() => {
+    if (capture?.speciesId) {
+      const currentSpecies = () => {
+        return speciesList.find((species) => species.id === capture.speciesId);
+      };
+      const speciesData = currentSpecies();
+      setSpeciesName(speciesData.name);
```

Comment:
Need to check that `speciesData` is not `null` first.

Reply:
Thank you, that makes sense. I'll try to fix it.

Reply:
We also need to clear the species name if there is no match, otherwise it may display the previous capture's species name.

Reply:
Thank you very much, I'm still learning the clean-up functions and it was a good learning experience. I think I got  it right this time.